addons/mail/static/tests/new/messaging_menu/notification_group_tests.js

```diff
@@ -0,0 +1,334 @@
+/** @odoo-module **/
+
+import { afterNextRender, start, startServer } from "@mail/../tests/helpers/test_utils";
+import { getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+let target;
+
+QUnit.module("notification group", {
+    async beforeEach() {
+        target = getFixture();
+    },
+});
+QUnit.test("notification group basic layout", async function (assert) {
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv["mail.channel"].create({});
+    const mailMessageId1 = pyEnv["mail.message"].create({
+        message_type: "email",
+        model: "mail.channel",
+        res_id: mailChannelId1,
+        res_model_name: "Channel",
+    });
+    pyEnv["mail.notification"].create([
+        {
+            mail_message_id: mailMessageId1,
+            notification_status: "exception",
+            notification_type: "email",
+        },
+        {
+            mail_message_id: mailMessageId1,
+            notification_status: "exception",
+            notification_type: "email",
+        },
+    ]);
+    const { click } = await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    assert.containsN(target, ".o-mail-notification-item", 2);
+    assert.containsN(target, ".o-mail-notification-item-name", 2);
+    assert.strictEqual(
+        $(target).find(".o-mail-notification-item-name")[1].innerText.trim(),
+        "Channel"
+    );
+    assert.strictEqual($(target).find(".o-mail-notification-item-counter").text().trim(), "(2)");
+    assert.containsN(target, ".o-mail-notification-item-date", 2);
+    assert.strictEqual($(target).find(".o-mail-notification-item-date")[0].innerText.trim(), "now");
+    assert.strictEqual($(target).find(".o-mail-notification-item-date")[1].innerText.trim(), "now");
+    assert.containsN(target, ".o-mail-notification-item-inlineText", 2);
+    assert.strictEqual(
+        $(target).find(".o-mail-notification-item-inlineText")[1].innerText.trim(),
+        "An error occurred when sending an email"
```

Comment:
Test is hard to read because it relies on `[0]`, `[1]`, etc.
Would be best to specifically target element from their visual clues, like with `:contains`